### DISCUSSION TITLE
Noobaa K8s API requests: >800'000 each day

### DISCUSSION THREAD

**Discussion by vinzent** [noobaa/noobaa-operator#1305] [General]
We have 3 Noobaa instances (part of ODF 4.13).  While analyzing K8s API Audit-logs we discovered that the ServiceAccount `system:serviceaccount:openshift-storage:noobaa` is the one with the most API requests. 

stats from 2024-02-21: 

* cluster a: 825'272 requests of 22'497'559 total
* cluster b: 876'862 requests of 17'703'618
* cluster c: 893'150 requests of 13'407'534

Given cluster c has probably not seen any noobaa resource change at all, the number of requests seems way too high.

nooba-core-0 and nooba-operator-* are using the noobaa serviceaccount. Probably it's the operator with the 2'000 reconciles each hour that emits the huge amount of requests?

**Comment by vinzent** [comment]
I suspect the PR https://github.com/noobaa/noobaa-operator/pull/1291 which links https://github.com/noobaa/noobaa-operator/issues/1282 might solve this.

**Comment by liranmauda** [comment]
Thanks for the input.
Closing 🙂 

Feel free to reopen or open a new one if needed.
